changelogs/fragments/353-docs-add-influxdb-flux-example.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

```suggestion
trivial:
```
I would prefer trivial, because it's not an user affecting change.